### ISSUE TITLE
Use `process.cwd()` instead of `__dirname`

### ISSUE THREAD

**Issue by fengyuanchen** [steveblue/rollup-plugin-inline-postcss#12]
https://github.com/steveblue/rollup-plugin-inline-postcss/blob/9e11867db5a7ad0ddbd3023d0c53cd9d14a1c386/index.ts#L32

---

User `__dirname`:

```js
console.log(findup.sync(__dirname, 'postcss.config.js'));
// ~/my-project/node_modules/rollup-plugin-inline-postcss/postcss.config.js
```

User `process.cwd()`:

```js
console.log(findup.sync(process.cwd(), 'postcss.config.js'));
// ~/my-project/postcss.config.js
```

**Comment by steveblue** [comment]
This change will be in the next release, currently on master.

**Comment by steveblue** [comment]
Fixed in 3.0.0
